src/main/java/at/esque/kafka/topics/converters/IntegerPropertyConverter.java

```diff
@@ -0,0 +1,20 @@
+package at.esque.kafka.topics.converters;
+
+import com.opencsv.bean.AbstractBeanField;
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+
+public class IntegerPropertyConverter extends AbstractBeanField {
+
+    @Override
+    public Object convert(String value) {
+        return new SimpleIntegerProperty(Integer.parseInt(value));
```

Comment:
Wie verhält sich diese Variante bei ungültigen Integer-Werten im CSV-File (auch im Vergleich zu Variante 1)?

Reply:
Variante 2:
"asdf" als partition im csv-file führt zu einer "java.lang.NumberFormatException"

Variante 1:
"asdf" als partition im csv-file führt zu "com.opencsv.exceptions.CsvDataTypeMismatchException: Konvertierung von asdf in int fehlgeschlagen."

Reply:
Solang der User in der Fehlermeldung sieht, bei welchem Wert das Problem aufgetreten ist, sollte das ausreichend sein.